packages/proxy/src/system.ts

```diff
@@ -108,3 +106,27 @@ export function getNetworkAliasForMac(networkName: string) {
 
   return cleanedName;
 }
+
+// TODO: make a better impl if you feel like it but for now this works
+export async function unsetProxy(): Promise<void> {
+  return new Promise((resolve, _) => {
+    getActiveInterface((err, { desc: interfacesName }) => {
+      if (err) {
+        console.error(err);
+        return;
+      }
+
+      // TODO: Cross Platform Support, and error handling.
+      if (platform() === "darwin") {
+        // make sure the interface is right because reasons
+        const networkInterfaceName = getNetworkAliasForMac(interfacesName);
+
+        const command = `networksetup -setautoproxyurl "${networkInterfaceName}" "off"`;
+        execSync(command);
+
+        console.log(`Proxy removed from [${interfacesName}]`);
```

Comment:
Do we not want some dope emoji? Or COLOR output?! (It's fine)

Reply:
i do actually! good call